Description: Fix 'allows to' typo, replace with 'allows <verb>ing'.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://trac.osgeo.org/grass/ticket/2704
Applied-Upstream: https://trac.osgeo.org/grass/changeset/65576

--- a/gui/wxpython/vdigit/g.gui.vdigit.html
+++ b/gui/wxpython/vdigit/g.gui.vdigit.html
@@ -43,7 +43,7 @@ A new vector map can be easily created f
 selecting &quot;New vector map&quot; in &quot;Select vector map&quot;
 combobox. A new vector map is created, added to the current layer tree
 in Layer Manager and opened for editing. &quot;Select vector map&quot;
-combobox in the digitizer toolbar also allows to easily switch between
+combobox in the digitizer toolbar also allows easily switching between
 vector maps to be edited.
 
 <h3>EDITING AN EXISTING VECTOR MAP</h3>
--- a/doc/parms_flags.txt
+++ b/doc/parms_flags.txt
@@ -64,7 +64,7 @@ Import:
 Export:
 *  input=name file=name for all raster, sites and vector modules
 *  export modules should *all* have an output= parameter, however they
-   should allow to specify "-" to write to stdout (for unix piping)
+   should allow specifying "-" to write to stdout (for unix piping)
 
 -> file is external file. Should be read from current directory, not
    within mapset!
--- a/gui/wxpython/docs/wxGUI.nviz.html
+++ b/gui/wxpython/docs/wxGUI.nviz.html
@@ -296,7 +296,7 @@ attributes.
 
 <h4>Cutting planes</h4>
 
-Cutting planes allow to cut surfaces along a plane. You can switch 
+Cutting planes allow cutting surfaces along a plane. You can switch 
 between six planes; to disable cutting planes switch to <em>None</em>.
 Initially the plane is vertical, you can change it to horizontal by
 setting <em>tilt</em> 90 degrees. The <em>X</em> and <em>Y</em> values
--- a/gui/wxpython/web_services/widgets.py
+++ b/gui/wxpython/web_services/widgets.py
@@ -964,7 +964,7 @@ class LayersList(TreeListCtrl, listmix.L
         return sel_layers_dict
 
     def OnListSelChanging(self, event):
-        """Do not allow to select items, which cannot be requested from server.
+        """Do not allow selecting items, which cannot be requested from server.
         """
         def _emitSelected(layer):
             title = layer.GetLayerData('title')
--- a/lib/ogsf/gs2.c
+++ b/lib/ogsf/gs2.c
@@ -1711,7 +1711,7 @@ int GS_load_att_map(int id, const char *
 	tbuff = gs_get_att_typbuff(gs, att, 1);
 
 	/* TODO: Provide mechanism for loading certain attributes at
-	   specified sizes, allow to scale or cap, or scale non-zero */
+	   specified sizes, allow scaling or capping, or scale non-zero */
 	if (ATT_MASK == att) {
 	    atty = ATTY_MASK;
 	}
@@ -3343,7 +3343,7 @@ int GS_load_3dview(const char *vname, in
 /*!
    \brief Init viewpoint
 
-   \todo allow to set center?
+   \todo allow setting center?
  */
 void GS_init_view(void)
 {
--- a/raster/r.null/r.null.html
+++ b/raster/r.null/r.null.html
@@ -36,7 +36,7 @@ Note that value is restricted to integer
 <p><em>r.null</em> and reclassified maps:<br>
 The problem is, if r.null was run on the reclass raster it would alter the
 original and any other reclass rasters of the original.  Therefore r.null 
-doesn't allow to recode reclassified maps (products of r.reclass).
+doesn't allow recoding reclassified maps (products of r.reclass).
 <br>
 So, the way to recode such a map is: The user makes a raster out of the
 reclass that isn't a reclass by copying it:<br>
--- a/temporal/t.rast.accdetect/t.rast.accdetect.html
+++ b/temporal/t.rast.accdetect/t.rast.accdetect.html
@@ -24,7 +24,7 @@ The <b>minimum</b> and <b>maximum</b> va
 process can be set, either by using space time raster datasets or 
 by using fixed values for all raster cells and time steps. 
 <p>
-Using space time raster datasets allow to specify minimum and maximum 
+Using space time raster datasets allow specifying minimum and maximum 
 values for each raster cell and each time step. For example, we want to 
 detect the germination (minimum value) and harvesting (maximum value) 
 dates for different crops in Germany using the growing-degree-day (GDD) 
